app/src/main/res/values/strings.xml

```diff
@@ -418,4 +418,7 @@
     <string name="lbl_manual_login">Manual Login</string>
     <string name="lbl_playback_control_ends">Ends at %1$s</string>
     <string name="watch_count_overflow">99+</string>
+    <string name="lbl_stars">Stars</string>
+    <string name="lbl_tomatoes">Tomatoes</string>
+    <string name="pref_default_rating">Default Ratings Scale</string>
```

Comment:
```suggestion
    <string name="pref_default_rating">Default rating type</string>
```

Could apply this consistently in the code too

Reply:
Changed all references of "RatingProvider" or any variation of "RatingProvider" to "RatingType". I assume that is what you meant here.